src/NuGet.Clients/PackageManagement.UI/Xamls/PackageItemControl.xaml

```diff
@@ -90,8 +120,58 @@
           TextWrapping="NoWrap"
           AutomationProperties.AutomationId="id"
           Text="{Binding Id}" />
+
+        <Grid
+          Grid.Row="1">
+          <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="auto" />
+            <ColumnDefinition Width="auto" />
+            <ColumnDefinition Width="auto" />
+            <ColumnDefinition Width="auto" />
```

Comment:
would be nice to provide comments on what each column represents. These are pretty hard to track